### ISSUE TITLE
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage

### ISSUE THREAD

**Issue by ghostsquad** [vivaxy/gacp#43]
I just received this warning when installing GACP.

```
$ npm i -g gacp
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

**Comment by vivaxy** [comment]
`gacp` depends on [`@commitlint/load`](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/load).
I think `@commitlint/load` had removed core-js@<3 in [this commit](https://github.com/conventional-changelog/commitlint/commit/2549f79698f672569ecfeb622c511eb1561054a4#diff-09a13c6999e8c9b42efb3203b14b6709). It's not released now. I am waiting for this commit to be released to resolve this issue.

**Comment by vivaxy** [comment]
Fixed in gacp@2.10.0
